consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/CMCConsumerGateway.java

```diff
@@ -38,22 +58,49 @@ public void individualRequest(IIndividualRequest individualRequest,
                                   long receiveTimeOut) throws OutageException, TimeoutException {
         LOGGER.debug("Invoke cmc target application service for individual request");
         try {
-            BreathingSpaceRequest request = xmlToObject.convertXmlToObject(individualRequest.getRequestPayload(),
-                                                                           BreathingSpaceRequest.class);
-            BreathingSpaceResponse response = breathingSpace.breathingSpace(request);
-            individualRequest.setRequestStatus(response.getProcessingStatus().name());
+            if (RequestType.CLAIM_STATUS_UPDATE.getRequestType().equals(individualRequest.getRequestType())) {
+                ClaimStatusUpdateRequest claimStatusUpdateRequest = null;
+                claimStatusUpdate.claimStatusUpdate(claimStatusUpdateRequest, "", "");
+            } else if (RequestType.BREATHING_SPACE.getRequestType().equals(individualRequest.getRequestType())) {
+                BreathingSpaceRequest request = xmlToObject.convertXmlToObject(individualRequest.getRequestPayload(),
+                        BreathingSpaceRequest.class);
+                BreathingSpaceResponse response = breathingSpace.breathingSpace(request);
+                individualRequest.setRequestStatus(response.getProcessingStatus().name());
+            }
         } catch (Exception e) {
             throw new CMCException(e.getMessage(), e);
         }
     }
 
     @Override
-    public void submitQuery(ISubmitQueryRequest submitQueryRequest,
+    public Object submitQuery(ISubmitQueryRequest submitQueryRequest,
                             long connectionTimeOut,
                             long receiveTimeOut) throws OutageException, TimeoutException {
         LOGGER.debug("Submitting query to target application[{}], for customer[{}]",
                      submitQueryRequest.getTargetApplication().getTargetApplicationCode(),
                      submitQueryRequest.getBulkCustomer().getSdtCustomerId());
+
+        // extract fromDate and toDate from submitQueryRequest
+        String[] values = getClaimDefencesFields();
+        String fromDate = values[0];
+        String toDate = values[1];
+        ClaimDefencesResponse response =  claimDefences.claimDefences(fromDate, toDate);
+
+        List<Object> listObjects = Arrays.asList(response.getResults());
+
+        return listObjects;
+    }
+
+    private String[] getClaimDefencesFields() {
+
+        // get dates from request defence criteria.
+        String xmlContent = SdtContext.getContext().getRawOutXml();
```

Comment:
why this xml content is read from SdtContext.getContext()

request xml should be available in ISubmitQueryRequest

Reply:
Discussed and demonstrated through debug.